### PR TITLE
network/sfeed: Update for 1.6

### PR DESCRIPTION
**Pull request by isaackwy**

### CODE REVIEW

File: network/sfeed/sfeed.SlackBuild [hunk 1]
```diff
@@ -25,7 +25,7 @@
 cd $(dirname $0) ; CWD=$(pwd)
 
 PRGNAM=sfeed
-VERSION=${VERSION:-1.5}
+VERSION=${VERSION:-1.6}
```
Comment (aclemons) [inline network/sfeed/sfeed.SlackBuild:28]: This one needs some updates:

```
# sbopkglint ../packages/15.0/x86_64/network/sfeed/*txz
Using tests from /usr/share/sbo-maintainer-tools/sbopkglint.d
Installing ../packages/15.0/x86_64/network/sfeed/sfeed-1.6-x86_64-1_SBo.txz to /tmp/sbopkglint.Fxcb0z ...OK
Running test: 05-basic-sanity...
--- forbidden directory exists: usr/share/doc
FAILED
Running test: 10-docs...OK
Running test: 15-noarch...OK
Running test: 20-arch...OK
Running test: 25-lafiles...OK
Running test: 30-manpages...OK
Running test: 35-desktop...OK
Running test: 40-newconfig...OK
Running test: 45-doinst...OK
Running test: 50-icons...OK
!!! sfeed-1.6-x86_64-1_SBo.txz: 1 failures
```

**Review by aclemons — CHANGES_REQUESTED** [review]
Could you check the doc dir here please?

**Comment by isaackwy** [comment]
I fixed the issue.
What was going on was that sfeed built the DOCPREFIX directory at /usr/share/doc/sfeed by default.
I initially addressed this by removing the folder - i.e. `rm -r /usr/share/doc/sfeed'
However, your checks have demonstrated that the above approach was insufficient.
Therefore, I decided to change DOCPREFIX (in make install) to /usr/doc/sfeed-$VERSION

Please check my Slackbuild again.

**Comment by aclemons** [comment]
All good now. Staged to my branch

**Pull request closed without merging**